templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -6,8 +6,8 @@ contents: |
   # This service is used to move a physical NIC into OVS and reconfigure OVS to use the host IP
   Requires=openvswitch.service
   Wants=NetworkManager-wait-online.service
-  After=NetworkManager-wait-online.service openvswitch.service node-valid-hostname.service
-  Before=network-online.target kubelet.service crio.service
```

Comment:
Was dropping `Before=network-online.target` intentional?

How about adding  `ConditionPathExists=!/etc/ignition-machine-config-encapsulated.json` to this service instead?

And we should do that for `kubelet.service` and `crio.service` too probably - basically everything we don't want to run until we've pivoted, rather than implicitly relying on having the firstboot service initiate a reboot before they've started.

Reply:
>Was dropping Before=network-online.target intentional?

Yes, after `network-online` mcd-pull and mcd-firstboot started, so ovs-configuration would always happen after network-online anyway

> How about adding ConditionPathExists=!/etc/ignition-machine-config-encapsulated.json to this service instead?

Good idea, added that

Reply:
I think it wouldn't hurt to leave it, though? Just in case we shuffle things around again...

Reply:
Removed unnecessary After/Before/Wanted changed for now - this can be re-iterated later